### DISCUSSION TITLE
核爆発の無効化について | About disabling nuclear explosion

### DISCUSSION THREAD

**Discussion by Kai-Z-JP** [Kai-Z-JP/KaizPatchX#161] [Opinion]
KaizPatchではVersion 1.4.0で核爆発を無効化する予定です。
これについて何か意見のある方はここに返信をお願いします。

In version 1.4.0, I plan to disable nuclear explosions.
If you have any comments about this, please reply here.

**Comment by lazytspo** [comment]
please do not

**Reply by Trollebas** [reply]
Correct decision, I once spammed a 410 block and knocked down an entire building of 5 floors 1-335.

**Reply by lazytspo** [reply]
> Correct decision, I once spammed a 410 block and knocked down an entire building of 5 floors 1-335.

i agree
